x-pack/test/functional/apps/machine_learning/anomaly_detection/multi_metric_job.ts

```diff
@@ -75,6 +75,7 @@ export default function({ getService }: FtrProviderContext) {
     this.tags(['smoke', 'mlqa']);
     before(async () => {
       await esArchiver.load('ml/farequote');
+      await ml.api.createCalendar('wizard-test-calendar');
     });
 
     after(async () => {
```

Comment:
Calendar cleanup (deletion) should be added to the after method.

Reply:
https://github.com/elastic/kibana/pull/52269#discussion_r354393098

Reply:
Dedicated cleanup is not needed (it's done with the `.ml-*` indices cleanup).